content/doc/book/glossary/index.adoc

```diff
@@ -93,6 +93,9 @@ Label::
     User-defined text for grouping <<agent,Agents>>, typically by similar
     functionality or capability. For example `linux` for Linux-based agents or
     `docker` for Docker-capable agents.
+LTS::
+    A long-term support <<release, Release>> line of Jenkins products, becoming
+    available for downloads every 12 weeks. 
```

Comment:
```suggestion
    available for downloads every 12 weeks. 
    See link:/download/lts/[this page] for more info.
```

Reply:
I like the addition of the hyperlink to the detailed description.  Rather than "this page", could the text of the hyperlink be:

> See the link:/download/lts/[LTS release line] for more info.